README.md

```diff
@@ -44,36 +44,31 @@ Many commands have an optional argument of space-separated component name(s), if
 ### Compile
 
 ```shell
-# Build and watch all components, run the server to preview demo pages
-npm run dev
+# Build all components
+npm run build
 
-# Build and watch on one or more components, run the server to preview demo pages
-npm run dev [component-name(s)]
+# Build one or more components
+npm run build [component-name(s)]
 ```
 
+The build command can accept a few flags; for more details, use `npm run build -- --help`.
+
 ### Preview
 
 ```shell
-# Runs server process to preview files
+# Runs server process to preview files (does not build)
 npm start
 
+# Builds, sets up the watch, and runs server process to preview files
+npm run dev
+
 # Runs storybook preview tool
 npm run storybook
 
 # Open documentation
 npm run docs
 ```
 
-### Compile & Preview
```

Comment:
@zhawkins These are duplicative; they still exist but I thought I'd remove them from documentation to reduce visibility